.github/workflows/ci.yml

```diff
@@ -103,6 +152,72 @@ jobs:
         run: npm run js-api-spec -- --sassSassRepo ../language --sassPackage ../build/npm
         working-directory: sass-spec
 
+  # The versions should be kept up-to-date with the latest LTS Node releases.
+  # They next need to be rotated October 2021. See
```

Comment:
Should we rotate these now, or save that for a separate PR?

Reply:
Separate PR, I think.